bokeh/embed.py

```diff
@@ -34,15 +42,19 @@ def components(plot_object, resources):
     Args:
         plot_object (PlotObject) : Bokeh object to render
             typically a Plot or PlotContext
-        resources (Resources, optional) : BokehJS resources config
-
+        resources : Deprecated argument
     Returns:
         (script, div) : UTF-8 encoded
 
     '''
+    
+    if resources is not None:
+        warn('the ``resources`` argument is deprecated and will be removed in'
+             'a future version.', DeprecationWarning, stacklevel=2)
+    
```

Comment:
Is this the correct way to deprecate a function argument? 

Also, I was wondering ... does this mean that we'll not be thinking about bringing back the non-concatenated bokehjs? Because if we do, it might be better to just keep some of these in place?